.gitignore

```diff
@@ -0,0 +1,4 @@
+/obj
+bld*.inc
```

Comment:
What is this file?

Reply:
try run make in repo

Reply:
![image](https://github.com/golded-plus/golded-plus/assets/74080/72112cdd-309f-4565-bf6e-63b847ff9a9b)

Reply:
also my make failed with
```
/usr/bin/ld: ../lib/lnx/libhunspell.a(suggmgr.o): in function `SuggestMgr::SuggestMgr(char const*, int, AffixMgr*)':
/home/shtirlic/projects/golded-plus/goldlib/hunspell/suggmgr.cxx:38:(.text+0x62): undefined reference to `AffixMgr::get_encoding()'
/usr/bin/ld: /home/shtirlic/projects/golded-plus/goldlib/hunspell/suggmgr.cxx:39:(.text+0x6d): undefined reference to `get_current_cs(char const
*)'
```